apps/src/templates/projects/projectsModule.js

```diff
@@ -0,0 +1,30 @@
+/** @file Redux actions and reducer for the Projects Gallery */
+import {registerReducers} from '@cdo/apps/redux';
+
+// Actions
```

Comment:
According to http://redux.js.org/docs/basics/Actions.html these are "Action types".